### PR TITLE
Add a resx translation guide

### LINKED ISSUE #4834: Write translation guide for Resources.resx files
Let's write a guide to help translators translate our `Resources.resx` files.

This guide is gonna be used in the issue and PR templates as a reference.

The PR to close this issue needs to be approved by both @amulware and @maritoggl .

### PR DESCRIPTION
**Pull request by daividssilverio**
## What's this?
This adds a little bit of documentation on how to approach the `Resources.resx` file when working on a translation for our app.

### Relationships
Closes #4834

## Why do we want this?
To have some documentation to give to translators so that they can work on our app.

## How is it done?
It adds the `docs/internationalization/how-to-contribute-with-translations.md` doc.
I tried to be as clear and direct to the point as possible.

### Why not another way?
Obviously, these docs won't work for everybody because you'd need to be at a little bit comfortable about using github, using text editors, that kind of stuff.

I think this is good enough for a start, but we could create something, fancy or not, that would parse our `Resources.resx` xml file and create something more user friendly with a web interface or something like that. But, even if we do that, we can still keep this piece of doc for people who'd wanna do it that way.

## :squid: Permissions
🦑 i18n team

### CODE REVIEW

File: docs/internationalization/how-to-contribute-with-translations.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <value>題</value>
+        <!-- You don't need to translate the comments -->
+        </data>
+    <data name="StopButton" xml:space="preserve">
+        <value>時辰儀止めるボタン</value>
+    </data>
+</root>
+```
+
+If you are translating into another language, the only difference is the language code in the `.resx` file. For example, a Brazilian Portuguese translation would require a file named `Resources.pt-Br.resx`, for Portugal Portuguese, it would be `Resources.pt-PT.resx`, for Latvian or Lettish, it would be `Resources.lv.resx`.
```
Comment (amulware) [inline docs/internationalization/how-to-contribute-with-translations.md]: Maybe specify where to look up the correct language code?

**Comment by amulware** [comment]
Looks fine to me overall, but can you give me a bit more context? Will this be linked from a larger guide? I assume we'll be organising translation issues in some project or something and there will be an overarching document explaining how that works, and that might link to this one?

**Comment by daividssilverio** [comment]
> Looks fine to me overall, but can you give me a bit more context? Will this be linked from a larger guide? I assume we'll be organizing translation issues in some project or something and there will be an overarching document explaining how that works, and that might link to this one?

Yes, this will be part of the issue "Write localization framework doc" #4839, which should connect all the dots.

File: docs/internationalization/how-to-contribute-with-translations.md [hunk 2]
```diff
@@ -66,3 +66,9 @@ The `Toggl.Shared` folder should now contain both files, the original `Resources
 ```
 
 If you are translating into another language, the only difference is the language code in the `.resx` file. For example, a Brazilian Portuguese translation would require a file named `Resources.pt-Br.resx`, for Portugal Portuguese, it would be `Resources.pt-PT.resx`, for Latvian or Lettish, it would be `Resources.lv.resx`.
+
+
+### Where can I find the language codes for Resources.`language code`.resx files? (pt-BR, pt-PT, lv, ja etc)
+More information on where to get the right language codes can be found in the [Microsoft .NET CultureInfo docs](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo?view=netframework-4.8#culture-names-and-identifiers).
+
+TLDR; "The CultureInfo class specifies a unique name for each culture, based on RFC 4646. The name is a combination of an [ISO 639 two-letter lowercase](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) culture code associated with a language and an [ISO 3166 two-letter uppercase](https://en.wikipedia.org/wiki/List_of_ISO_3166_country_codes) subculture code associated with a country or region".
```
Comment (amulware) [inline docs/internationalization/how-to-contribute-with-translations.md]: If it's always a combination, what about `ja`?

Comment (amulware) [reply docs/internationalization/how-to-contribute-with-translations.md]: Ans is there no list of all valid codes? I can see people getting confused about how to assemble the right one. Might not be a big issue, but the easier we make it for people, the better.

Or can we just create the files for new translations ourselves and have people add new strings until they are done?

**Comment by maritoggl** [comment]
No new comments on this, Sir Paul has covered the concerns I had. Looks fine overall.

File: docs/internationalization/how-to-contribute-with-translations.md (hunk 2 above)
Comment (daividssilverio) [reply docs/internationalization/how-to-contribute-with-translations.md]: > If it's always a combination, what about ja?

Actually, it doesn't have to always be a combination.

> Or can we just create the files for new translations ourselves and have people add new strings until they are done?

I honestly think that's easier to create a branch, fire up a PR for the users with the base file and let them work from there. What do you think?

File: docs/internationalization/how-to-contribute-with-translations.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+## Translation Guide
+
+Helping to add a translation to the Toggl app is very simple!
```
Comment (dshr) [inline docs/internationalization/how-to-contribute-with-translations.md]:
```suggestion
Adding a new language to the Toggl mobile app is very simple!
```

File: docs/internationalization/how-to-contribute-with-translations.md (hunk 2 above)
Comment (amulware) [reply docs/internationalization/how-to-contribute-with-translations.md]: I thought about committing an empty file to develop and letting them work on that. Branches make things complicated.

Comment (daividssilverio) [reply docs/internationalization/how-to-contribute-with-translations.md]: I don't think I follow how would that work.
Do you mean?
- When starting a new translation, we create the new `Resources.xx.resx` with a copy of the original file through our regular PRs (quickly as possible), ending with it on develop.
- Then what would the users/translators do with that?

**Comment by dshr** [comment]
All good, the contribution instructions bit makes it even clearer, which is always a good thing!

**Comment by amulware** [comment]
I don't have time to review further, please proceed without me.

**Pull request merged**